catalog/spatial/csw/spatial-csw-source-common/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/reader/TestGetRecordsMessageBodyReader.java

```diff
@@ -214,7 +185,7 @@ public void testGetMultipleMetacardsWithForeignText() throws Exception {
                 is);
         List<Metacard> metacards = cswRecords.getCswRecords();
         assertThat(metacards, not(nullValue()));
-        assertThat(metacards.size(), equalTo(10));
+        assertThat(metacards.size(), equalTo(1));
     }
 
     // //////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
What is this thing anyway? I don't think you touched it, but it should probably be cleaned up.